invokeai/app/api/routers/models.py

```diff
@@ -55,30 +55,28 @@ async def list_models() -> ModelsList:
     models = parse_obj_as(ModelsList, { "models": models_raw })
     return models
 
-# Kent wrote the below code. It is highly suspect, and should be reviewed. Once all issues have been identified and fixed, this comment can be removed.
-# Seriously, check this. I have no idea how to test it.
-
-""" Add Model """
+#Update Model
 @models_router.post(
     "/",
-    operation_id="add_model",
-    responses={201: {"model": Union[CkptModelInfo, DiffusersModelInfo], "new_model_list": ModelsList}},
+    operation_id="update_model",
+    responses={
+        201: {
+        "model": Union[CkptModelInfo, DiffusersModelInfo], 
+        "new_model_list": ModelsList
+        },
+        202: {
+        "description": "Model submission is processing. Check back later."
+        }, 
+    },
 )
-async def add_model(
-    model_info: Union[CkptModelInfo, DiffusersModelInfo],
-) -> Union[CkptModelInfo, DiffusersModelInfo]:
-    """Adds a new model"""
+async def update_model(
+    model_request: CreateModelRequest
+) -> CreateModelRequest:
+    #Adds a new model
```

Comment:
This should be triple-quoted to show up in the API docs (launch and go to `http://localhost:9090/docs` to see what the API docs will look like for your endpoints).